.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules/
 .next/
-env.sh
```

Comment:
You might want to keep both of these entries! Some people use `.env`, and others use a `env.sh` file.